src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -100,6 +100,21 @@ public static int createMonitoringTopicIfNotExists(String zkUrl, String topic, i
   }
 
   /**
+   * @param zkUrl zookeeper connection url
```

Comment:
Can you remove this method since it is not used anywhere? Maybe you can simply revert the change on this and other files that doesn't have 'effective' change so that the pull request can be cleaner.